<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1274</width>
    <height>920</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Unilectric PDP</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow,
QMenuBar,
QMenuBar::item,
QStatusBar,
QProgressBar,
QGroupBox,
/*QFrame,*/
QSpinBox,
QLineEdit,
QCheckBox,
QMessageBox,
QLineEdit,
QRadioButton,
QPushButton,
QCheckBox,
QLabel,
/*QWidget,*/
QTabWidget,
QListWidget,
QTabBar::tab,
QPlainTextEdit
{
    color: white;
	font-family: Courier New;
	font-size: 10pt;
	font-weight: bold;
	background-color: black;
}





/* QTab Widget */

QTabWidget::pane { /* The tab widget frame */
    border: 1px solid;
    border-color: #aaaaaa;
}
QTabBar::tab {
    border: 1px solid;
    border-color: #aaaaaa;
    background: solid;
    background-color: #444444;
   /* border-top-left-radius: 8px;
    border-top-right-radius: 8px;*/
    min-width: 30ex;
    padding-left: 16px;
    padding-right: 16px;

}
QTabBar::tab:hover {
    border: 1px solid;
    border-color: #ffffff;
    background-color: black;
    padding-left: 0px;
    padding-right: 0px;
}
QTabBar::tab:selected {
    font-size: 12pt;
    border: 1px solid;
    border-color: #ffffff;
    background-color: black;
    padding-left: 0px;
    padding-right: 0px;
}
QTabBar::tab:!selected {
    margin-top: 4px; /* make non-selected tabs look smaller */
}

/* QGroupBox */


/* QRadioButton */

QRadioButton::!checked{
    background:transparent;
}
QRadioButton::hover{
    background:#444444;
}
QRadioButton::checked{
    background:#444444;
}

 
QLabel {
	
	font-size: 10pt;

}

QSpinBox {
    border: 1px solid;
    border-color: white;
	font-size: 12pt;
}
QSpinBox::hover{
    background-color: #444444;
}



QLineEdit {
    border: 1px solid;
    border-color: white;
    background-color: black;
}
QLineEdit::hover{
    background-color: #444444;
}




QPushButton { 
	border: 1px solid;
    /*border-radius: 2px;*/
	border-color: #aaaaaa;
    background: solid;
    background-color: black;
     min-height: 20px;
}
QPushButton:!pressed {
    background-color: #444444;
}
QPushButton:hover {
    background: transparent;
}
QPushButton:disabled {
    background: transparent;
}
QPushButton:checked {
    background-color: #aa0000;
}
QPushButton:pressed {
    background-color: #888888;
}



QStatusBar 
{
	font-size: 10pt;
	background-color: #444444;
}
        
QProgressBar 
{
	color: #444444; /* if same as bkg color, hides text */
        font-size: 10pt;
}


QMenuBar 
{
    font-size: 10pt;
    background-color: #444444;
}

QMenuBar::item 
{
    spacing: 10px; /* spacing between menu bar items */
    padding: 1px 20px; /* height width */
    background: transparent;
    border-radius: 1px;
}

QMenuBar::item:selected 
{ 
}

QMenuBar::item:pressed 
{
}





</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>951</width>
      <height>711</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>771</width>
       <height>541</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>170</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RMS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>230</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Peak</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>350</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>410</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Min</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_rms">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>190</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_min">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>430</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_max">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>370</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_peak">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>250</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>171</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string> Relay </string>
     </property>
     <widget class="QRadioButton" name="radioButton_relayOn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>On</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_relayOff">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_read_relay">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_read_AC">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>60</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Read AC</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ctdc">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>310</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>290</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CT-DC</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_read_DMA">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>20</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Read DMA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_channel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_read_AC_filtered">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Read AC</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>301</width>
      <height>871</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>951</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="listWidget_comms_list">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_comms_test">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_comms_open">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_icon">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>10</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_write_command">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Write Command</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_read_status">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Read Status</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_relaySelect">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>5</y>
       <width>211</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string> Relay </string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
